search-service/src/main/kotlin/com/egm/stellio/search/entity/web/EntityHandler.kt

```diff
@@ -208,8 +223,9 @@ class EntityHandler(
                 QP.OPTIONS, QP.FORMAT, QP.COUNT, QP.OFFSET, QP.LIMIT, QP.ID, QP.TYPE, QP.ID_PATTERN, QP.ATTRS, QP.Q,
                 QP.GEOMETRY, QP.GEOREL, QP.COORDINATES, QP.GEOPROPERTY, QP.GEOMETRY_PROPERTY,
                 QP.LANG, QP.SCOPEQ, QP.CONTAINED_BY, QP.JOIN, QP.JOIN_LEVEL, QP.DATASET_ID,
+                QP.PICK // not implemented but needed for interoperability
```

Comment:
better add it and keep it in a specific commit, will be easier to remove after the Plug Test